test/e2e/framework/util.go

```diff
@@ -203,6 +204,12 @@ func GetMasterHost() string {
 	return masterURL.Hostname()
 }
 
+func skipInternalf(caller int, format string, args ...interface{}) {
```

Comment:
now we need to keep this function in this file

Reply:
It is better to write the reason why as `// NOTE` or something around this function.

Reply:
I see that this function is used within the framework/util.go at `RunIfContainerRuntimeIs` and `RunIfSystemSpecNameIs`.
What do you all think about moving these 2 functions to the e2eskipper pkg as well?
The name of these functions doesn't start with "skip" but other than that it has a similar behaviour.
And with those functions we can also move this out of the framework.

Reply:
good idea